<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChordGenerator</class>
 <widget class="QWidget" name="ChordGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chord Generator</string>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Noto Sans Black</family>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string> Uldarik's Chord Progression Generator</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Noto Sans Black</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Chords Randomizer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Select Numer of Bar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="bar_selector"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Select Tonality or enter chords for Custom Tonality</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="tonality_selector"/>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="chord_input_label">
           <property name="text">
            <string>Enter Chords (comma separated):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="chord_input_field"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="generate_btn">
         <property name="text">
          <string>Generate Chords</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>boogie.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Noto Sans Black</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Playback</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="play_btn">
           <property name="text">
            <string>Play Chords</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_btn">
           <property name="text">
            <string>Stop Playback</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Select Tempo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="tempo_selector"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="repetition_label">
             <property name="text">
              <string>Repetitions per Bar:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="repetition_selector"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Chose Midi Instrument/Port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refresh_midi_btn">
           <property name="text">
            <string>Refresh Devices List</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="midi_device_selector"/>
       </item>
       <item>
        <widget class="QGraphicsView" name="view">
         <property name="minimumSize">
          <size>
           <width>700</width>
           <height>0</height>
          </size>
         </property>
         <property name="backgroundBrush">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
